mobile/library/common/engine.h

```diff
@@ -82,6 +82,9 @@ class Engine : public Logger::Loggable<Logger::Id::main> {
    */
   envoy_status_t makeAdminCall(absl::string_view path, absl::string_view method, envoy_data& out);
 
+  /* Dump Envoy stats into the returned buffer*/
```

Comment:
drive-by nit: Looks like this should probably use `/** */` (javadoc-style) comments. Also, training ".", please. Is there a particular format that the stats are emitted in that we could document?